src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -351,11 +344,13 @@ uct_rc_mlx5_ep_bcopy_post(uct_rc_mlx5_ep_t *ep, unsigned opcode, unsigned length
                           /* RDMA */ uint64_t rdma_raddr, uct_rkey_t rdma_rkey,
                           int force_sig, uct_rc_iface_send_desc_t *desc)
 {
-    desc->super.sn = ep->tx.wq.sw_pi;
+    uint16_t sn;
```

Comment:
it's more efficient to remember sn inside desc immediately than to use 'sn' variable